app/electron/main.ts

```diff
@@ -78,6 +82,28 @@ const args = yargs(hideBin(process.argv))
   .help()
   .parseSync();
 
+const listPlugins = args['list-plugins'] === true;
+
+// If the user wants to list plugins, we don't need to start the app.
+if (listPlugins) {
+  const staticDir = path.join(process.resourcesPath, '.plugins');
+  const pluginsDir = path.join(app.getPath('userData'), 'plugins');
```

Comment:
Tried testing using PluginManager's `list` function, but it would break and return empty lists. I tried testing changes to `checkValidPluginFolder` to see if that would fix it but it caused new JS issues. I've made the other change and can continue testing with PluginManager but I seem to be hitting a wall with this

Reply:
Changed this to call execSync using the backend logic